CHANGELOG.md

```diff
@@ -114,4 +114,6 @@ can get data for both organization and organization brand pages.
 ## 3.0.4 (Apr 1, 2022)
 * Update `JsonUtils.getValue` method to handle JsonNumeric values correctly.
 
-## 3.0.5 (work in progress)
\ No newline at end of file
+## 3.0.5 (Apr 7, 2022)
+* Updated CommentAction.Attribute.value to use AttributedEntity, to fix issue where share 
+  requests with mentions aren't being persisted correctly.
```

Comment:
Please also add the `## 3.0.6 (work in progress)` line - I know I added that later last time but I am expecting a merge conflict because of that....best to do it here.